lib/routes/content.js

```diff
@@ -103,14 +118,92 @@ const routes = application => {
     });
 
     /*
-     *  Post to Content Type
-     */
-    app.post(`/${config.content.base}/:type/${config.content.actions.save}`, multipartMiddleware, (req, res, next) => {
+     * @name Individual Content Type Edit Page
+     *
+     * @param {object} req - HTTP Request
+     * @param {object} res - HTTP Response
+     * @param {object} next - Express callback
+    */
+    app.get(`/${config.content.base}/:type/:id/${config.content.actions.edit}`, (req, res, next) => {
```

Comment:
Maybe lots of metadata about that particular id? Like sunrise/sunset, publishable, workflow state, etc…?